package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@fulcrumapp/node-pg-format",
     "description": "Node.js implementation of PostgreSQL's format() to safely create dynamic SQL queries.",
-    "version": "0.0.1",
+    "version": "0.0.2",
```

Comment:
I don't think it really make sense to bump the version here if we're just updating dev dependencies?